eclass/nvidia-driver.eclass

```diff
@@ -212,21 +219,21 @@ nvidia-driver_check_kernel() {
 		ewarn "which are limited to the following kernels:"
```

Comment:
Above, I think you can do `${NV_KV_MAX_PLUS/./ }` (or with ver-cut 1-2 if it has more components). Also, `NV_KV_MAX_PLUS` is not documented anywhere.